### DISCUSSION TITLE
How many 🌳🌳 are there in the wood?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#844] [Galaxy Trivia] [answered]
In the local wood there are a number of trees. On these trees there are a number of birds. The local gamekeeper knows that there are as many birds on each tree as there are trees in the wood, and that there are between 2000 and 2100 birds in total.

**Comment by peterbaccaro** [comment]
There are **45** trees in the woods as 45 is the only number whose square lies between 2000 and 2100.

44 x 44 = 1936
**45 x 45 = 2025**
46 x 46 = 2116

**Comment by Carol42** [comment, marked as answer]
Since the square root of 2000 is approximately 44.72 and the square root of 2100 is approximately 45.82, it can be inferred that the number of birds per tree and the total number of trees in the wood fall within this range. Consequently, we can estimate that there are around 45 birds 🐦 on each tree🌳 and a total of 45 trees 🌳🌳 in the wood, resulting in a cumulative count of 2025 birds.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/Carol42?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. 🎊🎉

**Reply by Carol42** [reply]
Thanks! 😄
